README.md

```diff
@@ -10,7 +10,7 @@ Modelled as a state machine, with the help of AWS step functions, Trapheus resto
 </p>
 <p align="center"><a href="https://circleci.com/gh/intuit/Trapheus"><img src="https://circleci.com/gh/intuit/Trapheus.svg?style=svg" alt="TravisCI Build Status"/></a>
 <a href = "https://coveralls.io/github/intuit/Trapheus?branch=master"><img src= "https://coveralls.io/repos/github/intuit/Trapheus/badge.svg?branch=master" alt = "Coverage"/></a>
-  <a href="http://www.serverless.com"><img src="http://public.serverless.com/badges/v3.svg" alt="serverless badge"/></a>
+  <a href="https://www.serverless.com"><img src="https://public.serverless.com/badges/v3.svg" alt="serverless badge"/></a>
```

Comment:
Can we change the "src" url to ```https://github.com/intuit/Trapheus/assets/112308320/c7764b65-fb72-46ed-824b-c82c6374ec8b``` . The current https endpoint for serverless badge is broken so i have hosted the image in this new space.

Reply:
Sure I would do that